lib/config/presets/internal/helpers.ts

```diff
@@ -30,4 +30,15 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  pinGitHubActionDigestsToSemver: {
```

Comment:
Marking this PR as draft, and creating this PR review comment, to make sure we discuss @travi's comment first:

- https://github.com/renovatebot/renovate/pull/23663#issuecomment-1693806214

Reply:
to maybe clarify, my goal isnt to hold up this helper from moving forward. i think this helper would be a useful step, especially where i'd like to use it with teams that might not grasp the goal with the existing raw workaround. 

instead, i was just hoping to share some thoughts about where follow-up steps could maybe take this idea a bit further if the bigger picture could be captured. i thought that might also help to give some potentially different context to consider the unresolved questions from the other comment thread above.

Reply:
Maybe we should hold up this helper... 😉

I think the big point is that we should make Renovate easy to use, and easy to do the right thing. So we should think if adding a new preset is really the right thing to do. 😄 

Maybe making a breaking change to the `helpers:pinGitHubActionDigests` preset in `v37`, so it always "pins GitHub Actions digests to SemVer versions" is better than making a new `pinGitHubActionDigestsToSemverPreset`. I'd prefer for Renovate to just do the right thing out of the box. 😄 That way I don't have to search through a lot of config options to find what I want. 🙃

Reply:
I also found another issue that seems related:

- https://github.com/renovatebot/renovate/issues/7516